bedrock/src/transaction/contracts/erc20.rs

```diff
@@ -87,16 +86,12 @@ pub struct MetadataArg {
 impl Is4337Encodable for Erc20 {
     type MetadataArg = MetadataArg;
 
-    fn as_execute_user_op_call_data(&self) -> Bytes {
-        ISafe4337Module::executeUserOpCall {
-            // The token address
-            to: self.token_address,
-            value: U256::ZERO,
-            data: self.call_data.clone().into(),
-            operation: SafeOperation::Call as u8,
-        }
-        .abi_encode()
-        .into()
+    fn target_address(&self) -> Address {
```

Comment:
Better to define these fns on the Is4337Encodable instead of duplicating inside each impl?